tests/dags/providers/provider_api_scripts/test_cleveland_museum.py

```diff
@@ -35,197 +35,177 @@ def validate_url_string():
-    query_param = {"cc": 1, "has_image": 1, "limit": 1, "skip": -1}
-    with patch.object(clm.delay_request, "get", return_value=None) as mock_get:
-        response_json, total_images = clm._get_response(query_param)
-
-    assert response_json is None
-    assert total_images == 0
-    # Retries
-    assert mock_get.call_count == 3
-
-
-def test_handle_response():
-    response_json = _get_resource_json("handle_response_data.json")
-    data = response_json["data"]
-    actual_total_images = clm._handle_response(data)
-    expected_total_images = 100
-
-    assert actual_total_images == expected_total_images
+class TestClevelandDataIngester(unittest.TestCase):
+    def setUp(self):
+        self.clm = ClevelandDataIngester()
+        self.image_store = ImageStore(provider=prov.CLEVELAND_DEFAULT_PROVIDER)
+        self.clm.media_stores = {"image": self.image_store}
+
+    def _get_resource_json(self, json_name):
+        with open(RESOURCES / json_name) as f:
+            resource_json = json.load(f)
+            return resource_json
+
+    def test_build_query_param_default(self):
+        actual_param = self.clm.get_next_query_params({})
+        expected_param = {"cc": "1", "has_image": "1", "limit": 1000, "skip": 0}
+        assert actual_param == expected_param
+
+    def test_build_query_param_increments_offset(self):
+        previous_query_params = {"cc": "1", "has_image": "1", "limit": 1000, "skip": 0}
+
+        actual_param = self.clm.get_next_query_params(previous_query_params)
+        expected_param = {"cc": "1", "has_image": "1", "limit": 1000, "skip": 1000}
+        assert actual_param == expected_param
+
+    def test_get_image_type_web(self):
+        image_data = self._get_resource_json("image_type_web.json")
+        actual_image = self.clm._get_image_type(image_data)
+
+        expected_image = {
+            "url": "https://openaccess-cdn.clevelandart.org/1335.1917/1335.1917_web.jpg",
+            "width": "1263",
+            "height": "775",
+            "filesize": "716717",
+            "filename": "1335.1917_web.jpg",
+        }
+        assert actual_image == expected_image
+
+    def test_get_image_type_print(self):
+        image_data = self._get_resource_json("image_type_print.json")
+        actual_image = self.clm._get_image_type(image_data)
+
+        expected_image = {
+            "url": "https://openaccess-cdn.clevelandart.org/1335.1917/1335.1917_print.jpg",
+            "width": "3400",
+            "height": "2086",
+            "filesize": "5582485",
+            "filename": "1335.1917_print.jpg",
+        }
+
+        assert actual_image == expected_image
+
+    def test_get_image_type_full(self):
+        image_data = self._get_resource_json("image_type_full.json")
+        actual_image = self.clm._get_image_type(image_data)
+
+        expected_image = {
+            "url": "https://openaccess-cdn.clevelandart.org/1335.1917/1335.1917_full.tif",
+            "width": "6280",
```

Comment:
I think `width` and `height` should be ints. Or does the image store convert them anyways?

Reply:
They get [cast to int](https://github.com/WordPress/openverse-catalog/pull/555/files/61129690bee1a72a8587c226516af925d0c8274f#diff-5fa4a32973ff2b61bd0ce8750c964c488aa0c4613da88f6f76f807dab3099214R65) in `get_record_data` when we pass them to the ImageStore, yep! This is just testing `get_image_type`, which just selects which image type to use when parsing record data (preferring web, print, full in that order). What's being returned at this point is just json.